test/e2e_env/universal/timeout/meshtimeout.go

```diff
@@ -0,0 +1,98 @@
+package timeout
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/kumahq/kuma/test/e2e_env/universal/env"
+	. "github.com/kumahq/kuma/test/framework"
+)
+
+func PluginTest() {
+	meshName := "meshtimeout"
+	faultInjection := fmt.Sprintf(`
+type: FaultInjection
+mesh: "%s"
+name: fi1
+sources:
+   - match:
+       kuma.io/service: demo-client
+destinations:
+   - match:
+       kuma.io/service: test-server
+       kuma.io/protocol: http
+conf:
+   delay:
+     percentage: 100
+     value: 5s
```

Comment:
Does our echo / testserver not handle response delay? maybe we could use something like this: https://github.com/mendhak/docker-http-https-echo#add-a-delay-before-response ?